app/konnect/private-connections/aws-privatelink.md

```diff
@@ -37,123 +37,12 @@ Create a VPC, subnets, and a security group in AWS. For more information, see th
-| ------ | ------------ |
-| AP | com.amazonaws.vpce.eu-central-1.vpce-svc-0c3f0574080bdd859 | ap.svc.konghq.com |
-| EU | com.amazonaws.vpce.eu-central-1.vpce-svc-05e6822fbce58e1a0 | eu.svc.konghq.com |
-| IN | com.amazonaws.vpce.eu-central-1.vpce-svc-0a5c165336502e526 | in.svc.konghq.com |
-| ME | com.amazonaws.vpce.eu-central-1.vpce-svc-0e6497e6df9928a80 | me.svc.konghq.com |
-| US | com.amazonaws.vpce.eu-central-1.vpce-svc-01d3dd232e277feeb | us.svc.konghq.com |
-{% endnavtab %}
-
-{% navtab ap-southeast-2 %}
-| {{site.konnect_short_name}} Geographical Region | PrivateLink service name | Private DNS name |
-| ------ | ------------ |
-| AP | com.amazonaws.vpce.ap-southeast-2.vpce-svc-055ba6ff5a3f551c9 | ap.svc.konghq.com |
-| EU | com.amazonaws.vpce.ap-southeast-2.vpce-svc-02a339e8dc8ec72c6 | eu.svc.konghq.com |
-| IN | com.amazonaws.vpce.ap-southeast-2.vpce-svc-050ea149424be6d3c | in.svc.konghq.com |
-| ME | com.amazonaws.vpce.ap-southeast-2.vpce-svc-008f231c7501e72c2 | me.svc.konghq.com |
-| US | com.amazonaws.vpce.ap-southeast-2.vpce-svc-0600dd84f39e7b12a | us.svc.konghq.com |
-{% endnavtab %}
-
-{% navtab ap-southeast-1 %}
-| {{site.konnect_short_name}} Geographical Region | PrivateLink service name | Private DNS name |
-| ------ | ------------ |
-| AP | com.amazonaws.vpce.ap-southeast-1.vpce-svc-08f76d27d29b02a09 | ap.svc.konghq.com |
-| EU | com.amazonaws.vpce.ap-southeast-1.vpce-svc-0a04604ecaed18457 | eu.svc.konghq.com |
-| IN | com.amazonaws.vpce.ap-southeast-1.vpce-svc-0c6699c89ac27323c | in.svc.konghq.com |
-| ME | com.amazonaws.vpce.ap-southeast-1.vpce-svc-00689717c9085d08a | me.svc.konghq.com |
-| US | com.amazonaws.vpce.ap-southeast-1.vpce-svc-0eeaa22ed2d6268eb | us.svc.konghq.com |
-{% endnavtab %}
-{% endnavtabs %}
-{% endcapture %}
-{{ tabs | indent }}
-
-1. Verify the service. If the service name can't be verified, contact [Kong Support](https://support.konghq.com/support/s/).
-
-1. Once the service name is verified, select the VPC, subnets, and security groups to associate with this endpoint. Make sure that:
-  * The security group accepts inbound traffic on TCP port 443.
-  * The DNS name parameter in the additional settings is enabled.
-
-1. Create the VPC endpoint and wait for its status to change to available. We recommend waiting for 10 minutes before using it.
-
-1. Update your data plane configuration to connect to the control plane through the private connection. Use the private DNS name that matches your control plane geo in the tables above. For the US {{site.konnect_short_name}} geo, the updated Kong data plane configuration in `kong.conf` looks like this:
-```sh
-cluster_control_plane = us.svc.konghq.com/cp/{cluster_prefix}
-cluster_server_name = us.svc.konghq.com
-cluster_telemetry_endpoint = us.svc.konghq.com:443/tp/{cluster_prefix}
-cluster_telemetry_server_name = us.svc.konghq.com
-```
-{% endnavtab %}
-
-{% navtab Konnect Admin API %}
```

Comment:
@healthy-pod Is it on purpose that we're only displaying one option now, ie Control Plane <> Data Plane Communication, and no Konnect Admin API?

Reply:
no thanks for catching that

Reply:
updated